README.md

```diff
@@ -3,9 +3,33 @@
 > This is a work-in-progress document. Changes are tracked through PRs
 > and issues.
 
+**What's on this page:**
+
+  * [Introduction](#introduction)
+  * [Ideal functionality and trust assumptions in contact tracing systems](#ideal-functionality-and-trust-assumptions-in-contact-tracing-systems)
+  * [A strawman protocol](#a-strawman-protocol)
+  * [The CEN Protocol](#the-cen-protocol)
+  * [CEN sharing with Bluetooth Low Energy](#cen-sharing-with-bluetooth-low-energy)
+  * [Contributors](#contributors)
```

Comment:
```suggestion
  * [Contributors](#contributors)
As it is a work-in-progress, this page also contains rough notes: 
```

Reply:
Just a brief note to indicate to the reader that the first part and the second part are at different readiness levels.